lisp/ess-custom.el

```diff
@@ -646,211 +646,345 @@ regardless of where in the line point is when the TAB command is used."
 nil means to use R/S indentation.")
 (make-variable-buffer-local 'ess-indent-line-function)
 
-(defvar ess-indent-level 2
+(defvar ess-indent-level 4
```

Comment:
This change is a bit tricky. As you surely know, there is a special style DEFAULT which picks indentation settings from global variables and indents as such. The problem is that ESS had 2 space indentation. A couple of years ago we switched to 4 and users could set `ess-style` to DEFAULT to get the previous behavior.